doc/rados/operations/add-or-rm-osds.rst

```diff
@@ -216,190 +216,200 @@ of the following form:
 Starting the OSD
 ----------------
 
-After you add an OSD to Ceph, the OSD is in your configuration. However,
-it is not yet running. The OSD is ``down`` and ``in``. You must start
-your new OSD before it can begin receiving data. You may use
-``service ceph`` from your admin host or start the OSD from its host
-machine:
+After an OSD is added to Ceph, the OSD is in the configuration. However, until
```

Comment:
"in the configuration"? Maybe "In the cluster"

Reply:
Yeah, that is better. Accepted.